src/main/kotlin/blackjack/Round.kt

```diff
@@ -0,0 +1,9 @@
+package blackjack
+
+class Round(private val trump: Trump = Trump()) {
+    fun getCard(): Card = trump.getCard()
+
+    companion object {
+        const val TWENTY_ONE = 21
```

Comment:
```suggestion
        const val BLACKJACK = 21
```
블랙잭에서는 숫자 21은 "블랙잭"이라고 부릅니다.